src/string.rs

```diff
@@ -130,6 +162,25 @@ impl String {
     }
   }
 
+  /// Allocates a new string from UTF-16 data. Only returns an empty value when
+  /// length > kMaxLength.
+  pub fn new_from_two_byte<'s>(
+    scope: &mut HandleScope<'s, ()>,
+    buffer: &[u16],
+    new_type: NewStringType,
+  ) -> Option<Local<'s, String>> {
+    unsafe {
+      scope.cast_local(|sd| {
+        v8__String__NewFromTwoByte(
+          sd.get_isolate_ptr(),
+          buffer.as_ptr(),
+          new_type,
+          buffer.len() as int,
```

Comment:
I realize you copied this from `new_from_one_byte()` but a good follow-up change would be to turn the cast into a fallible operation, in case someone passes in a buffer > 2^31.